VideoWeb/VideoWeb.AcceptanceTests/Steps/HearingRoomSteps.cs

```diff
@@ -164,17 +172,25 @@ public void ThenTheJudgeCanSeeInterpreterAndInterpreteeOnParticipantList()
         {
             var interpreter = _c.Test.ConferenceParticipants.FirstOrDefault(x => x.HearingRole.ToLower() == "interpreter");
             var interpretee = _c.Test.ConferenceParticipants.FirstOrDefault(x => x.Id == interpreter.LinkedParticipants.FirstOrDefault().LinkedId);
-
-            _browsers[_c.CurrentUser].Driver.WaitUntilVisible(HearingRoomPage.InterPreterName(interpreter.DisplayName)).Text.Should().Contain($"{interpretee.DisplayName}");
+            
+            var interpreterText = _browsers[_c.CurrentUser].Driver
+                                    .WaitUntilElementExists(HearingRoomPage.ParticipantPanel)
+                                    .FindElement(HearingRoomPage.InterPreterName(interpreter.DisplayName)).Text.Trim();
```

Comment:
I think you can get rid if FindElement as the WaitUntil element will find it

Reply:
This is finding element within a parent, first WaitUntil is to find participant panel and second is to find interpreter div within participant panel.